src/CMakeLists.txt

```diff
@@ -329,6 +329,13 @@ if (ZSTD_LIBRARY)
     endif ()
 endif()
 
+set (LZMA_LIBRARY liblzma)
+set (LZMA_INCLUDE_DIR ${ClickHouse_SOURCE_DIR}/contrib/xz/src/liblzma/api)
```

Comment:
And I guess it is better to move this out into separate `contrib/xz-cmake/CMakeLists.txt`, and add an interface library.
One of similar files is `contrib/protobuf-cmake/CMakeLists.txt` (protobuf has it's own cmake rules, while in clickhouse there is just a wrapper that install some options)

Reply:
What should I do to pass description check?
I've added record to `CHANCHELOG.md`

Reply:
>What should I do to pass description check?

Update description of PR and include changelog entry using specified format.
There is template for PRs - https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/.github/PULL_REQUEST_TEMPLATE.md

Just copy it and modify the template, but leave the format.

Reply:
Yes, it's much better to provide our own CMakeLists.

CMake often cannot be safely reused (without building unneeded targets, polluting build options).